openedx/core/djangoapps/course_live/serializers.py

```diff
@@ -76,6 +79,9 @@ def update(self, instance: LtiConfiguration, validated_data: dict) -> LtiConfigu
         if lti_config.get('additional_parameters', None):
             instance.lti_config['additional_parameters'] = lti_config.get('additional_parameters')
 
+        if validated_data.get('lti_1p1_client_secret') == '':
```

Comment:
What is the use case for this?

Reply:
This is a case where the user saves an empty password, It will use an already saved password.